client/src/main/scala/drt/client/components/ForecastFileUploadPage.scala

```diff
@@ -42,16 +43,16 @@ object ForecastFileUploadPage {
 
   val component: Component[Props, Unit, Unit, CtorType.Props] = ScalaComponent.builder[Props]("ForecastFileUpload")
     .render_P { _ =>
-      val fileUploadStateRCP = SPACircuit.connect(m => FileUploadStateModel(m.fileUploadState))
+      val fileUploadStateRCP = SPACircuit.connect(m => FileUploadStateModel(m.fileUploadState, m.airportConfig))
       <.div(
         fileUploadStateRCP(fileUploadStateMP => {
           if (fileUploadStateMP().fileUploadState.isEmpty) {
-            upload
+            upload(fileUploadStateMP().airportConfig.head.portCode.iata)
```

Comment:
calling head on airport config is a bit risky here, if it's a POT it should have a `renderReady` method on it that you can use to ensure that you don't end up with an exception.